docs/source/setup.md

```diff
@@ -55,7 +55,7 @@ Don't forget to replace YOUR_GITHUB_USER_NAME with your git username.
 
 ### Step 3: Setup code base
 
-* Create a python virtual environment and install python dependencies.
+* Create a python virtual environment and install python dependencies
```

Comment:
Please don't remove full-stop from the end. I think the earlier version looks better.

Reply:
Have removed the full-stops.

Reply:
Please revert to earlier version i.e `dependencies.` in this line only.